src/conversion_helper.js

```diff
@@ -580,7 +580,36 @@ function preprocessComment(comment) {
     return input;
   }
 
+  function abbreviationProcessor(comment) {
+    const abbreviations = {
+      k: 1000,
+      thousand: 1000,
+      m: 1000000,
+      mill: 1000000,
+    };
+
+    const abbreviationsRegex = Object.keys(abbreviations).map(
```

Comment:
Instead of joining all the abbreviation regexes together, I think it would be clearer if we worked on each key individually. In psudocode:
```
comment = Object.keys(abbreviations).reduce((abbrRegex) => {
  //create number + abbrRegex
  //find matches in comment
  //replace with abbreviations[abbrRegex]
  //return new comment string with replaced abbreviations
}, comment);
```

Open to discussion though, if you disagree